docs/courses/robotics/3.1-pumps-and-pipettes.md

```diff
@@ -87,12 +114,23 @@ Based on https://github.com/ac-rad/digital-pipette
 - [Wire cutter/stripper combo](https://www.amazon.ca/Professional-crimping-Multi-Tool-Stripper-Multi-Function/dp/B073YG65N2)
 - [Mini flathead screwdriver set](https://www.amazon.ca/dp/B08QCT9NHY/) (one for regulators and a smaller one for PiCowbell terminals)
 
+Digital pipettes are the unsung heroes of precise liquid handling! 🦸‍♀️ They're essential in:
```

Comment:
"Digital Pipette" is actually somewhat specific to the design by Naruki and co. "Automated single-channel syringe pump" is probably the best generic term for Naruki's "Digital Pipette".

@n-yoshikawa feel free to weigh in if you have a moment.